app/client/models/error.js

```diff
@@ -1 +1 @@
-module.exports = (state, bus) => bus.on('error', err => { throw error })
```

Comment:
Finding stuff like this with a linter is really nice indeed. Also very nice to get rid of unused stuff.